lab3.md

```diff
@@ -424,7 +430,7 @@ Official Template: true
 Pull the template down:
 
 ```sh
-$ faas-cli template pull node10-express
+$ faas-cli template pull https://github.com/openfaas-incubator/node10-express-template
```

Comment:
Please change to `faas-cli template store pull`

Reply:
Ah ! Okay. I will update that.

Reply:
Updated it.